src/Data/Skills/act_dex.lua

```diff
@@ -3744,6 +3744,55 @@ skills["ExplosiveArrow"] = {
 			area = false,
 		},
 	},
+	explosiveArrowFunc = function(activeSkill, output, globalOutput, env)
+		if activeSkill.skillPart ~= 1 and activeSkill.skillPart ~= 2 then
+			-- This doesn't apply to the "Arrow" skill part. That works like a normal skill.
+			return
+		end
+
+		local modDB = env.modDB
+		local enemyDB = activeSkill.actor.enemy.modDB
+		local skillModList = activeSkill.skillModList
+		local duration = calcSkillDuration(skillModList, activeSkill.skillCfg, activeSkill.skillData, env, enemyDB)
+		local fuseLimit = skillModList:Sum("BASE", activeSkill.skillCfg, "ExplosiveArrowMaxFuseCount")
+		local activeTotems
+		if activeSkill.skillFlags.totem then
+			activeTotems = modDB:Override(nil, "TotemsSummoned") or skillModList:Sum("BASE", activeSkill.skillCfg, "ActiveTotemLimit", "ActiveBallistaLimit")
+		end
+
+		local barrageProjectiles = nil
+		if skillModList:Flag(nil, "SequentialProjectiles") and not skillModList:Flag(nil, "OneShotProj") and not skillModList:Flag(nil,"NoAdditionalProjectiles") and not skillModList:Flag(nil, "TriggeredBySnipe") then
+			barrageProjectiles = skillModList:Sum("BASE", skillCfg, "ProjectileCount")
```

Comment:
```suggestion
			barrageProjectiles = skillModList:Sum("BASE", activeSkill.skillCfg, "ProjectileCount")
```
Without specifying the config properly, you're missing 4 projectiles.